docs/src/main/asciidoc/security-getting-started-tutorial.adoc

```diff
@@ -43,10 +43,10 @@ To examine the completed example, download the {quickstarts-archive-url}[archive
 
 [source,bash,subs=attributes+]
 ----
-git clone {quickstarts-clone-url}
+git clone {quickstarts-clone-url} -b {github-branch}
```

Comment:
Please don't do that.

In the website, I use the following trick:
```
:quickstarts-clone-url: -b 3.8 https://github.com/quarkusio/quarkus-quickstarts.git
```
and that will be an issue with this change.

I think you could do the same. I define it in `_attributes-local.adoc`.

Reply:
To be thorough, here is the entire section:

```
:quickstarts-clone-url: -b 3.8 https://github.com/quarkusio/quarkus-quickstarts.git
:quickstarts-archive-url: https://github.com/quarkusio/quarkus-quickstarts/archive/3.8.zip
:quickstarts-blob-url: https://github.com/quarkusio/quarkus-quickstarts/blob/3.8
:quickstarts-tree-url: https://github.com/quarkusio/quarkus-quickstarts/tree/3.8
```

Reply:
Thanks for the clarification @gsmet. I should have looked up that attribute value.
cc: @michalvavrik

Reply:
Hi, @gsmet 

The current value in `asciidoc/_attributes.adoc` is:
```
:quickstarts-clone-url: {quickstarts-base-url}.git
```
Should we update it to the following?
```
:quickstarts-clone-url: -b {github-branch} {quickstarts-base-url}.git
```

Reply:
I've removed the change, above. Thanks.

Reply:
https://github.com/quarkusio/quarkus/issues/41817

Reply:
not sure if it's just me, but https://github.com/quarkusio/quarkus/issues/41817 is very technical. the gist of this change was: if you read 3.8 branch, guides suggest you to checkout and test main branch of quickstarts. right now, users are following 3.8 guides but using updated code on the main branch

Reply:
@michalvavrik are you talking about community? If so please be specific. 3.8 guides should point to the 3.8 branch as shown in https://quarkus.io/version/3.8/guides/validation .

Reply:
> @michalvavrik are you talking about community? If so please be specific. 

I was explained this line written by Rolfe:

> I created the following issue to reflect @michalvavrik's feedback 

And that he created #41817 because of me. I never requested this. I wanted that 3.8 branch of documentation referts to 3.8 of quickstarts. that wasn't true for a product and Rolfe wrote that this comes from the community and tried to change it here. But I don't see it in the 3.8 community.

This comment thread is about `git clone {quickstarts-clone-url} -b {github-branch}` on main, which is IMHO unnecessary if community 3.8 is correct.

In that light, I tried to say that I don't understand neither #41817 nor discussion above. As this comment thread and PR are closed, let's forget about it.

Thank you

Reply:
One of the QE reviewers requested a change in the community `/asciidoc` files that had been synchronized to the product repository. I was following through to ensure this request didn't get overlooked. Upon inspecting the community attributes files, I didn't see an affordance for the version number. When I reviewed the community `3.12` docs, I noticed that the links pointed to `main` instead of `3.12`. However, had I inspected the `3.8` docs, I would have seen that the links correctly pointed to `3.8`. Thank you for the conversation.